Scripts/Runtime/Entities/TaskSystem/Job/JobConfig/AbstractJobConfig.cs

```diff
@@ -275,6 +275,32 @@ public IJobConfigRequirements RequireCDFEForWrite<T>()
 
             return this;
         }
+        
+        //*************************************************************************************************************
+        // CONFIGURATION - REQUIRED DATA - DynamicBuffer
+        //*************************************************************************************************************
+        
+        /// <inheritdoc cref="IJobConfigRequirements.RequireDBFEForRead{T}"/>
+        public IJobConfigRequirements RequireDBFEForRead<T>()
+            where T : struct, IBufferElementData
+        {
+            DynamicBufferAccessWrapper<T> wrapper = new DynamicBufferAccessWrapper<T>(AccessType.SharedRead, TaskSystem);
+            AddAccessWrapper(new JobConfigDataID(typeof(DynamicBufferAccessWrapper<T>.DynamicBufferType), Usage.Read),
```

Comment:
Discussed on call. Maybe we should have a token type in abstract wrapper so that all of the wrappers consistently report their type vs some using the raw data type and others some wrapped internal token.

With a comment on this that it's used for a token for job config.